analysers/analyser_osmosis_highway_vs_building.py

```diff
@@ -503,7 +505,10 @@ def analyser_osmosis_full(self):
         self.run(sql30.format("", ""), self.callback30)
         self.run(sql31.format("", ""), self.callback31)
         self.run(sql32.format("", ""), self.callback32)
-        self.run(sql40.format("", ""), self.callback40)
+        if self.country and self.country == "CA-BC": # Too many results
```

Comment:
Use `startswith` to be region split ready.

Reply:
Good point, done!